pkg/slo-controller/noderesource/plugins/midresource/plugin.go

```diff
@@ -161,6 +186,13 @@ func (p *Plugin) calculate(strategy *configuration.ColocationStrategy, node *cor
 	}
 	memory := resource.NewQuantity(allocatableMemory, resource.BinarySI)
 
+	// add unallocated
+	unallocated := p.getUnallocated(node, podList)
+	// CPU need turn into milli value
+	unallocatedCPU, unallocatedMemory := resource.NewQuantity(unallocated.Cpu().MilliValue(), resource.DecimalSI), unallocated.Memory()
```

Comment:
Please consider individual parameters in the ColocationConfig to over-commit the Prod unallocated resources. Since the Prod pods may schedule onto the node anytime, the Prod unallocated resources are often rather unstable for Mid pods than the reclaimable resources.

Reply:
done.